test/cppsuite/tests/test_live_restore.cpp

```diff
@@ -111,15 +111,15 @@ class database_model {
 
 static const int iteration_count_default = 2;
 /*
- * FIXME-WT-13825: Set thread_count_default to non zero once extent list concurrency is implemented.
```

Comment:
I feel like we can do this now independent of removing the `fill_holes_on_close` config.